app/frontend/courses/curricula/components/CoursesCurriculum__SubmissionsAndFeedback.res

```diff
@@ -51,17 +51,13 @@ let submissionStatusIcon = (~passed) => {
   let text = passed ? tr("completed") : tr("rejected")
   let color = passed ? "green" : "red"
 
-  <div className="max-w-fc">
-    <div
-      className={"flex justify-center border-2 rounded-lg border-" ++ (color ++ "-500 px-4 py-6")}>
-      {passed
-        ? <div className="fa-stack text-green-500 text-lg">
-            <i className="fas fa-certificate fa-stack-2x" />
-            <i className="fas fa-check fa-stack-1x fa-inverse" />
-          </div>
-        : <i className="fas fa-exclamation-triangle text-3xl text-red-500 mx-1" />}
-    </div>
-    <div className={"text-center text-" ++ (color ++ "-500 font-bold mt-2")}> {text |> str} </div>
+  <div className="max-w-fc flex items-center">
```

Comment:
What is ` max-w-fc `?

Reply:
`max-w-fc` was a custom class name when the max-width fit content wasn't available in Tailwind. Now, we can replace it with 'max-w-fit'.